config/i18n-tasks.yml

```diff
@@ -116,6 +117,7 @@ ignore_unused:
   - 'gobierto_plans.plan_types.show.read_more'
   - 'gobierto_admin.layouts.application.modules.*'
   - 'gobierto_admin.layouts.application.global'
+  - 'gobierto_data.data_type.*'
```

Comment:
Why ignored?

Reply:
Because it doesn't find they are being used in the JS.